src/irsync.cpp

```diff
@@ -169,53 +169,53 @@ usage() {
         "iRODS collection foo2.",
         " ",
         "The command compares the checksum values and file sizes of the source",
-        "and target files to determine whether synchronization is needed. Therefore,",
-        "the command will run faster if the checksum value for the specific iRODS file,",
+        "and target locations to determine whether synchronization is needed. Therefore,",
+        "the command will run faster if the checksum value for the specific iRODS data object,",
         "no matter whether it is a source or target, already exists and is registered",
-        "with iCAT. This can be achieved by using the -k or -K options of the iput",
-        "command at the time of  ingestion, or by using the ichksum command after the",
+        "in the catalog. This can be achieved by using the -k or -K options of the iput",
+        "command at the time of ingestion, or by using the ichksum command after the",
         "data have already been ingested into iRODS.",
 
-        "If the -s option is used, only the file size (instead of the the size and",
+        "If the -s option is used, only the file size (instead of the size and",
         "checksum value) is used for determining whether synchronization is needed.",
-        "This mode gives a faster operation but the result is less accurate.",
+        "This mode provides a faster operation but the result provides less assurance.",
         " ",
         "The command accepts multiple sourceFiles|sourceDirectories and a single",
-        "targetFile|targetDirectory. It pretty much follows the syntax of the UNIX",
-        "cp command with one exception- irsync of a single source directory to a ",
-        "single target directory. In UNIX, the command:",
+        "targetFile|targetDirectory. It follows the syntax of the UNIX",
+        "cp command with one exception - irsync of a single source directory to a ",
+        "single target collection. In UNIX, the command:",
         " ",
         "     cp -r foo1 foo2",
         " ",
-        "has a different meaning depending on whether the target directory foo2 ",
-        "exists. If the target directory exists, the content of source directory foo1",
-        "is copied to the target directory foo2/foo1. But if the target directory",
-        "does not exist, the content is copied to the target directory foo2.",
+        "has a different meaning depending on whether the target directory foo2",
+        "already exists. If the target directory exists, the content of source directory",
+        "foo1 is copied into the target directory foo2/foo1. But if the target directory",
+        "foo1 does not already exist, the content is copied as the target directory foo2.",
```

Comment:
`foo1` is identified as source directory and target directory across these four lines.

Seems it should only be referred to as the source directory?

Reply:
oh... i think there's a bug in the words.   looking again...

Reply:
yep, bug found - good eye.   updated in second commit.